server/utils.js

```diff
@@ -464,44 +465,47 @@ function dataToSchema(listOfElements) {
 }
 
 function fixAddress(schemaObject, geoObject) {
+  // The returned object will be POST/PATCH(ed),
```

Comment:
Not really misleading, but what it does is, replace the address with the one received from the geocoder. As the addresses are not really well formatted and using upperCase in the json from openEuskadi, I decided to made this kind of 'enrichment'. Regarding the `keyValues` option, this function is called to add Restaurants into Orion, so we need to add them using the `value` field as agreed.
Still, if you think it is not worth it, I can remove it and keep the address as they are